lib/internal/backend/repository/repository.go

```diff
@@ -149,7 +149,7 @@ func (rb *RepositoryBackend) supportsRegistry(indexURL string, version registryV
 		defer res.Body.Close()
 	}
 	if err != nil || !ok {
-		if rb.insecure {
+		if rb.insecure.Http {
```

Comment:
We should use "HTTP": https://github.com/golang/go/wiki/CodeReviewComments#initialisms